### PR TITLE
Adds support for ''' style triple quotes

### LINKED ISSUE #49: How to use  ''' instead of """  ?
Hello, I intend to use ' over " , but I don't know how to do .
Support both would be better,

### PR DESCRIPTION
**Pull request by Modelmat**
This works as the quoteStyle is passed from where it is detected in
baseFactory throughout all the commands which detect the docstring
opener and quotes. These include

 - validEnterActivation
 - docstringIsClosed
 - isMultiString
 - contentText

I haven't properly updated the unit testing to test for `'''` quotes, as it
is a very tedious operation that I didn't feel like doing.

This fixes #49

### CODE REVIEW

**Comment by Modelmat** [comment]
I've just noticed that CircleCI has failed some tests, I'll see what I can do to fix that tomorrow if it hasn't already been done.

**Comment by NilsJPWerner** [comment]
Thanks @Modelmat, I will review once you've fixed the tests  👍

**Comment by Modelmat** [comment]
Those are fixed. Do you wish the unit tests to be updated to check with both `'''` and `"""`, or is just `"""` fine? I'm not sure how to do it without a large amount of duplicated code.

**Comment by Modelmat** [comment]
What's hapenning with this?

**Comment by NilsJPWerner** [comment]
Hey @Modelmat, I've been in the process of moving across the country and haven't much time to check github. Will take a look later today.

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Visual Studio Code extension to quickly generate docstrings for python functions
 ## Usage
 Cursor must be on the line directly below the definition to generate full auto-populated docstring
 
-* Press enter after opening docstring with triple quotes (""")
+* Press enter after opening docstring with triple quotes (""" or ''')
```
[the remark this answers is not in the document]
Comment (Modelmat) [reply README.md]: Why wouldn't it?
I thought I designed this to work with either, depending on how it's entered-  `"""` then hitting enter. The quoteStyle setting is for only when the menu command is invoked, the automatic docstring auto-detects the style used to open it.

File: src/extension.ts [hunk 2]
```diff
@@ -9,7 +9,9 @@ export function activate(context: vs.ExtensionContext): void {
         vs.commands.registerCommand(
             'extension.generateDocstring', () => {
                 let editor = vs.window.activeTextEditor;
-                let autoDocstring = new AutoDocstring(editor);
+                let config = vs.workspace.getConfiguration("autoDocstring");
```
[the remark this answers is not in the document]
Comment (Modelmat) [reply src/extension.ts]: I have set this as default - it is used when the type of quote is passed in. (I use a this to pass it through when hitting enter on quotes)

**Pull request merged**